public/app/features/explore/LiveLogs.tsx

```diff
@@ -146,19 +146,19 @@ class LiveLogs extends PureComponent<Props, State> {
         >
           {this.rowsToRender().map((row: LogRowModel) => {
             return (
-              <div className={cx(logsRow, styles.logsRowFade)} key={row.uid}>
+              <td className={cx(logsRow, styles.logsRowFade)} key={row.uid}>
```

Comment:
This now generates a warning because `<td>` is nested in `<div>`:
```
Warning: validateDOMNesting(...): <td> cannot appear as a child of <td>.
    in td (created by LiveLogs)
    in td (created by LiveLogs)
    in div (created by LiveLogs)
    in div (created by LiveLogs)
    in LiveLogs (created by Context.Consumer)
    in WithTheme(undefined) (created by LiveTailControls)
    in LiveTailControls (created by LogsContainer)
    in div (created by Collapse)
    in div (created by Collapse)
    in Collapse (created by LogsContainer)
    in Transition (created by CSSTransition)
    in CSSTransition (created by LogsCrossFadeTransition)
    in LogsCrossFadeTransition (created by LogsContainer)
    in LogsContainer (created by ConnectFunction)
    in ConnectFunction (created by HotExportedConnect(LogsContainer))
    in AppContainer (created by HotExportedConnect(LogsContainer))
    in HotExportedConnect(LogsContainer) (created by AutoSizer)
    in ErrorBoundary (created by ErrorBoundaryAlert)
    in ErrorBoundaryAlert (created by AutoSizer)
    in main (created by AutoSizer)
    in div (created by AutoSizer)
    in AutoSizer (created by Explore)
    in div (created by Explore)
    in div (created by Explore)
    in Explore (created by ConnectFunction)
    in ConnectFunction (created by HotExportedConnect(Explore))
    in AppContainer (created by HotExportedConnect(Explore))
    in HotExportedConnect(Explore) (created by Wrapper)
    in ErrorBoundary (created by ErrorBoundaryAlert)
    in ErrorBoundaryAlert (created by Wrapper)
    in div (created by Wrapper)
    in div (created by Scrollbars)
    in div (created by Scrollbars)
    in Scrollbars (created by CustomScrollbar)
    in CustomScrollbar (created by Wrapper)
    in div (created by Wrapper)
    in Wrapper (created by ConnectFunction)
    in ConnectFunction (created by HotExportedConnect(Wrapper))
    in AppContainer (created by HotExportedConnect(Wrapper))
    in HotExportedConnect(Wrapper) (created by LoadableComponent)
    in LoadableComponent (created by Component)
    in Component (created by Component)
    in ThemeProvider (created by Component)
    in Component (created by ConfigProvider)
    in ConfigProvider
    in ErrorBoundary (created by ErrorBoundaryAlert)
    in ErrorBoundaryAlert
    in Provider
```

Shouldn't there be a `<tr>` instead of `<div>` now?

Reply:
Yes, right! Fixed.